README.md

```diff
@@ -1 +1,106 @@
-# auditor-node
\ No newline at end of file
+# Running auditor node
+Running auditor node is as simple as filling one config file and calling a `docker-compose up -d` command.
+## Preparing all data
+### Platform-dependent data
+Request platform administrator the following information:
+
+    - NETWORK_PASSPHRASE   - This is the identificator of the network
+    - MASTER_ACCOUNT_ID    - This is the master account of the platform
+    - BASE_EXCHANGE_NAME   - This is the human-readable name ot the platform
+    - Gateway url          - URL:PORT of the publicly-visible core
+    - History buckets' URL - this is the url for downloading history of the platform
+    - PUBLIC_KEYs of publicly-visible cores
+
+You should receive something like this (in free format):
+```
+NETWORK_PASSPHRASE: "Tutorial network!"
+MASTER_ACCOUNT_ID:  "GBA4EX43M25UPV4WIE6RRMQOFTWXZZRIPFAI5VPY6Z2ZVVXVWZ6NEOOB"
+BASE_EXCHANGE_NAME: "TokenD"
+Gateway url: "abd2c72d477b411e984f20aeae5344b9-7951a3850f76c4e7.elb.us-east-1.amazonaws.com:8090"
+PUBLIC_KEYS: ["GDEM3RHX6XZVIN23XM3UT44ILBYXP5P4BO6R4MV3US7PEER232RSIXCB", "GBC3SSF7AS4J3SU3FYKXZUKNMAMUOIROGP3ZNY4WJL4WJBFVU66FFEJO"]
+History buckets' URL: "https://buckets.tutorial.tokend.io"
+```
+
+### Generate
+Generate `NODE_SEED` by running following command:
+```
+docker run --rm --entrypoint=core tokend/core:3.2.1 --genseed
+```
+You should see the following output:
+```
+Using DEPRECATED command-line syntax.
```

Comment:
maybe not use deprecated cli?

Reply:
Difficult to find non-deprecated syntax
during 5 mins I didn't find, so it may take some time

Reply:
https://www.stellar.org/developers/stellar-core/software/commands.html
just `gen-seed`